app/models/post.rb

```diff
@@ -5,8 +5,12 @@ class Post < ApplicationRecord
 
   after_save :update_posts_counter
 
-  def return_recent_comments
-    comment.order(created_at: :desc).limit(5)
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :likes_counter, comparison: { greater_than_or_equal_to: 0 }
+  validates :comments_counter, comparison: { greater_than_or_equal_to: 0 }
```

Comment:
- [ ] kindly add validation for like and comment-counter being an integer

Reply:
Noted.